README.md

```diff
@@ -200,9 +208,9 @@ Refer [above section](#create-an-instance--reference-before-using) on how to cre
   Saves `data` in specific `key` in sessionStorage. If the key is not provided, the library will warn.
   Following types of JavaScript objects are supported:
 
-  |   Parameter   |        Description                                   |
-  | ------------- | ---------------------------------------------------- |
-  |     config     |     For opening a new tab i.e. URL and windowName    |
+  | Parameter | Description                                   |
```

Comment:
This (and the remaining table formatting) was automatically done by prettier (sorry for the noise).